src/encoded/static/components/cart/index.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * This file lets other files import this directory to get the cart Redux reducer function and any
+ * cart-related rendering components.
+ *
+ * You'll see references to terms used for various kinds of carts:
+ *
+ * active - The in-memory representation of the cart held in a Redux store
+ * shared - The cart in a user's `carts` object referenced by "/carts/<uuid>"
+ * saved - The cart contents in a user's `carts` object
+ *
+ * "active" carts hold saved (if logged in) and unsaved (if logged out) elements. "shared" carts
+ * hold saved elements. Users who aren't logged in can only have an "active" cart. "shared" carts,
+ * when displayed with the cart's uuid, can be shared with others.
+ */
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import _ from 'underscore';
+import {
+    ADD_TO_CART,
+    ADD_MULTIPLE_TO_CART,
+    REMOVE_FROM_CART,
+    REMOVE_MULTIPLE_FROM_CART,
+    CACHE_SAVED_CART,
+    CART_OPERATION_IN_PROGRESS,
+    NO_ACTION,
+} from './actions';
+import cartAddElements from './add_elements';
+import CartAddAll from './add_multiple';
+import cartCacheSaved from './cache_saved';
+import CartBatchDownload from './batch_download';
+import CartClear from './clear';
+import cartSetOperationInProgress from './in_progress';
+import CartMergeShared from './merge_shared';
+import cartRemoveElements from './remove_elements';
+import cartSave from './save';
+import CartSearchControls from './search_controls';
+import CartShare from './share';
+import CartStatus from './status';
+import CartToggle from './toggle';
+import { mergeCarts } from './util';
+
```

Comment:
Some of these methods are camelCase and some are PascalCase - is because there's a difference in the types of methods?

Reply:
The PascalCase ones are React components while the camelCase ones are regular functions.